src/docs/content/getting-started.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Getting Started
```

Comment:
Should we rename this to usage? Or at least link Installation to it

Reply:
Yeah I think usage is a better name, I was thinking we add a more intuitive "quickstart" to this page that goes through a journey of starting with a basic uncontrolled component, and then moving onto some control, customization, etc.!